src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/DetailControl.xaml.cs

```diff
@@ -204,5 +204,19 @@ private void ExecuteUserAction(UserAction action, NuGetActionType actionType)
                     }
                 });
         }
+
+        public void Dispose()
+        {
+            Dispose(disposing: true);
+            GC.SuppressFinalize(this);
+        }
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                _packageMetadataControl.Dispose();
```

Comment:
My understanding is that WPF should dispose this child automatically. Did you find that not to be the case?

Reply:
It's been a while since I implemented that but I think it wasn't properly disposing of the _markdownPreview property in the PackageMetadataReadMeControl control